### PR TITLE
chore: submission guidelines

### LINKED ISSUE #19: project submission guidelines
For when we start accepting projects that people have created to be showcased on our website, we need to come up with some clear and concise acceptance guidelines and criteria.

### PR DESCRIPTION
**Pull request by zaida04**
closes #19

### CODE REVIEW

File: SUBMISSION_GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Fiveable Open Source Project Submission Guidelines
+
+- Projects must be constructive and unique, repos that are intended to serve as a personal website or documentation websites are not allowed. The exception to this are external guides for libraries that don't provide a guide. 
+- Projects must have a clear CODE_OF_CONDUCT.md that clearly denounces any discriminatory, unwelcoming, or any otherwise harmful behavior that damages/hinders the positive environment we've strived to create.
```
Comment (exu3) [inline SUBMISSION_GUIDELINES.md]: Should we link an example code of conduct? https://www.contributor-covenant.org/version/2/0/code_of_conduct/ perhaps

File: SUBMISSION_GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	- repo description 
+	- clear instructions as to how to setup a local environment for contributors
+	- (encouraged) gifs/photos/text demonstrating example usage
+	- (encouraged) demo
+	- repo structure explanation (if isn't self-explanatory)
+- Projects must be contributable, cannot be archived or privated, and must be licensed under an appropriate license that is Open Source foundation certified. You can find an abiding license at (https://choosealicense.com/)
+- Projects must be submitted by either the repo owner or someone with the "maintainer" status on said repo. "contributor" status does not meet this criteria.
+- If tht well with our environment. Toxic or unwelcoming behavior is not tolerated at all.e project is a library/API, its routes/methods must be documented to a reasonable extent.
+- Submitter must display behavior that will fit well with our environment. Toxic or unwelcoming behavior is not tolerated at all.
+- If the project is a fork, it must be transformative to an extent (??).
```
Comment (exu3) [inline SUBMISSION_GUIDELINES.md]: Maybe we could evaluate these types of projects on a case-by-case basis. Also the `(??)`

**Review by exu3 — CHANGES_REQUESTED** [review]

File: SUBMISSION_GUIDELINES.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	- repo description 
+	- clear instructions as to how to setup a local environment for contributors
+	- (encouraged) gifs/photos/text demonstrating example usage
+	- (encouraged) demo
+	- repo structure explanation (if isn't self-explanatory)
+- Projects must be contributable, cannot be archived or privated, and must be licensed under an appropriate license that is Open Source foundation certified. You can find an abiding license at (https://choosealicense.com/)
+- Projects must be submitted by either the repo owner or someone with the "maintainer" status on said repo. "contributor" status does not meet this criteria.
+- If tht well with our environment. Toxic or unwelcoming behavior is not tolerated at all.e project is a library/API, its routes/methods must be documented to a reasonable extent.
+- Submitter must display behavior that will fit well with our environment. Toxic or unwelcoming behavior is not tolerated at all.
+- If the project is a fork, it must be transformative to an extent. This will be decided on a case by case basis
```
Comment (exu3) [inline SUBMISSION_GUIDELINES.md]: small thing, but can you add a period at the end please? 😄 (line 15)

**Review by exu3 — APPROVED** [review]
Thanks!

**Review by sarthaktexas — APPROVED** [review]

**Pull request merged**